cmd/tesla-keygen/main.go

```diff
@@ -57,7 +57,20 @@ func printPublicKey(skey protocol.ECDHPrivateKey) bool {
 	if err != nil {
 		return false
 	}
-	pem.Encode(os.Stdout, &pem.Block{Type: "PUBLIC KEY", Bytes: derPublicKey})
+
+	// If outputFile is provided, write to file, else write to stdout
+	var out io.Writer = os.Stdout
+	if outputFile != "" {
```

Comment:
Currently `usageText` says:

```
The program writes the public key to stdout (except when deleting a key)
```

Please change to

```
The program writes the public key to stdout (except when deleting a key or setting the output location with -output)
``

Reply:
Suggestion applied